pkg/cmd/render/render.go

```diff
@@ -342,12 +303,14 @@ func (r *renderOpts) Run() error {
 		return err
 	}
 
-	if err := r.manifest.ApplyTo(&templateData.ManifestConfig); err != nil {
-		return err
+	// Set the template dir only if not already set to support
+	// overriding the path in unit tests.
+	if len(r.templateDir) == 0 {
```

Comment:
I might miss it and can be a follow-up but do we print the write path during render?

Reply:
This is the input path, and it is not written. There's no way to set it other than in the code, though. The full output path is logged e.g.

```
Writing asset: /tmp/assets-316323197/manifests/etcd-metric-signer-secret.yaml
```